pkg/apis/pingcap/v1alpha1/tidbclusterautoscaler_types.go

```diff
@@ -149,6 +149,43 @@ type TidbMonitorRef struct {
 	Name string `json:"name"`
 }
 
-// TODO: sync status
+// BasicAutoScalerSpec describes the basic status for auto-scaling
+type BaseAutoScalerStatus struct {
+	// name is the name of the given metric
+	Name string `json:"name,omitempty"`
```

Comment:
The metrics property in AutoScaler is a list. If we need to show the metrics value, we should make the metrics status as list.

Reply:
Currently, We use one metric to calculate the autoscaling. So i think it is just show the metric in effect.